packages/wallet-frontend/lib/wallet.js

```diff
@@ -0,0 +1,11 @@
+/* global harden */
+
+// TODO: Don't just make this an adapter.
```

Comment:
yeah, I think the `vat-foo` file should only be used by swingset, and both `vat-foo` and this file should import a `buildRootObject` from the same (other) place.

Reply:
I think I'll reserve cleanups for our new wallet implementation, rather than this one.

My plan after this is to get the new wallet merged, then to move this old wallet out into a separate `agoric-labs` repo (which will be a terrific negative-code day!).